numpy/random/tests/test_random.py

```diff
@@ -1515,8 +1523,8 @@ def setup(self):
     def check_function(self, function, sz):
         from threading import Thread
 
-        out1 = np.empty((len(self.seeds),) + sz)
-        out2 = np.empty((len(self.seeds),) + sz)
+        out1 = np.empty((len(self.seeds), ) + sz)
+        out2 = np.empty((len(self.seeds), ) + sz)
```

Comment:
I guess `(x,)` is an exception to the space after comma rule. Let me clarify that to "space after comma not before a closing group, like `]` or `)`"

Reply:
Fixed.
Shows that I should check [the standard](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements) before deciding matters of taste preference.